src/authutils/token/fastapi.py

```diff
@@ -0,0 +1,105 @@
+from asyncio import Future, get_event_loop
+from collections import OrderedDict
+
+import httpx
+from fastapi import Security, HTTPException
```

Comment:
fastapi is an optional install, right? I guess it's fine to assume if someone is trying to use token.fastapi that they installed fastapi